### PR TITLE
completed css-positioning

### PR DESCRIPTION
**Pull request by jdfenlason**
Screen Shot 2021-03-12 at 2 03 07 PM
Screen Shot 2021-03-12 at 2 21 12 PM
Screen Shot 2021-03-12 at 2 26 23 PM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  bottom: 0px
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+position: absolute;
+top:0px;
+bottom: 100px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:40]: This can be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work!! One minor fix:

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -35,9 +35,7 @@
 
 .overlay {
   background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
-position: absolute;
-top:0px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css]: Oh dear, you still needed these lines 😅

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-positioning/absolute.css (hunk 2 above)
Comment (jdfenlason) [reply css-positioning/absolute.css]: Blonde moment my bad

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**